### PR TITLE
Fix typos in contributing docs.

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
Let me know if these fixes are ok, so I can continue to review other places.

### CODE REVIEW

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -42,11 +42,11 @@ The topic title, which is the first line of the document, is the only level 1 (
 === Section Title
 ----
 
-== Topic File Names
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: Our style guide says that "file name" needs to be two words unless it's a variable.

File: contributing_to_docs/doc_guidelines.adoc [hunk 2]
```diff
@@ -171,10 +171,10 @@ how it will render across any branches that the content appears in.
 
 Do not use markup in headings.
 
-*Do not use internal company server names in command or example output*. See suggested host name examples https://docs.openshift.com/container-platform/latest/install_config/install/planning.html#multi-masters-using-native-ha-colocated[here].
+*Do not use internal company server names in command or example output*. See suggested hostname examples https://docs.openshift.com/container-platform/latest/install_config/install/planning.html#multi-masters-using-native-ha-colocated[here].
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: Revert this one - host name needs to be two words.

File: contributing_to_docs/doc_guidelines.adoc [hunk 3]
```diff
@@ -490,7 +490,7 @@ This renders as:
 Use the `GET` operation to do x.
 
 === Command Syntax and Examples
-The main distinction between showing command syntax and example is that a command syntax should just show customers how to use the command without real values. An example on the other hand should show the command with actual values with an example output of that command, where applicable.
+The main distinction between showing command syntax and example is that a command syntax should just show customers how to use the command without real values. An example, on the other hand, should show the command with actual values with an example output of that command, where applicable.
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: s/example/an example

File: contributing_to_docs/doc_guidelines.adoc [hunk 4]
```diff
@@ -514,11 +514,11 @@ $ oc get <object_type> <object_id>
 ----
 
 ==== Examples
-As mentioned an example of a command should use actual values and also show an output of the command, as shown in the following example. In some a heading may not be required.
+As mentioned an example of a command should use actual values and also show an output of the command, as shown in the following example. In some, a heading may not be required.
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: s/As mentioned/As mentioned,
s/some/some examples
s/may/might

File: contributing_to_docs/doc_guidelines.adoc [hunk 5]
```diff
@@ -95,7 +95,7 @@ If you have a section heading that you do not want to appear in the TOC (like if
 http://asciidoctor.org/docs/user-manual/#discrete-or-floating-section-titles
 
 A discrete heading also will not get a section number in the Customer Portal
-build of hte doc. Previously, we would use plain bold mark-up around a heading
+build of hte doc. Previously, we would use plain bold markup around a heading
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: s/hte/the

File: contributing_to_docs/doc_guidelines.adoc [hunk 6]
```diff
@@ -479,7 +479,7 @@ $ oc get endpoints --all-namespaces --template \
 === Inline Code or Commands
 Do NOT show full commands or command syntax inline within a sentence. The next section covers how to show commands and command syntax.
 
-Only use case for inline commands would be general commands and operations, without replaceables and command options. In this case an inline command is marked up using the back ticks:
+The only use case for inline commands would be general commands and operations, without replaceables and command options. In this case, an inline command is marked up using the backticks:
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc]: s/would be/is
s/the backticks/backticks

**Review by kalexand-rh — COMMENTED** [review]
Hey @alexandrevicenzi, I have some comments on your changes.

**Comment by alexandrevicenzi** [comment]
@kalexand-rh thanks for the feedback.

**Comment by kalexand-rh** [comment]
@alexandrevicenzi, thanks for making updates and squashing. :) I think your changes look good now, so I'll merge.

**Pull request merged**